de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/MetadataTemplateViewDialog.java

```diff
@@ -49,45 +51,54 @@
  * Created by sriram on 5/9/16.
  */
 public class MetadataTemplateViewDialog extends IPlantDialog {
-   final MetadataView.Appearance appearance = GWT.create(MetadataView.Appearance.class);
+    final MetadataView.Appearance appearance = GWT.create(MetadataView.Appearance.class);
```

Comment:
This dialog is pretty huge, it can probably be split into a separate dialog and view class.